### ISSUE TITLE
disass: call addresses in disassembly are relative, should be absolute if possible

### ISSUE THREAD

**Issue by xs-exp** [pszostek/cp#16]
Example:
disass: 

```
401434: call 0xfffffffffffffdec
```

objdump:

```
  401434:       e8 e7 fd ff ff          callq  401220 <__libc_start_main@plt>
```

observations:

```
(uint64)0 - 0xfffffffffffffdec = 0x214
0x1434-0x214 = '0x1220'
```

**Comment by xs-exp** [comment]
Related to #27?

**Comment by pszostek** [comment]
Fixed with e8c1afe53bc999b913883f3756aaa9e95c30e230, tested with 0cecec2
